src/cortxfs/cortxfs_fh.c

```diff
@@ -97,6 +97,11 @@ struct cfs_fh_serialized {
 	cfs_ino_t ino_num;
 };
 
+struct cfs_fs *cfs_fs_from_fh(const struct cfs_fh *fh)
+{
+	return fh->fs;
```

Comment:
Added the `dassert` on `fh` and `cfs_fh_invariant` in commit f6ebea2a7cb64472c4f5a13e5e834c64d906082c.
Note: Have not added dassert on `cfs_fh_invariant` in `cfs_fh_stat` as invariant is again calling this API so it will be a never ending recursive call.